awd/cli.py

```diff
@@ -181,43 +181,47 @@ def deposit(
     try:
         s3_client.client.list_objects_v2(Bucket=bucket)
     except ClientError as e:
-        logger.error(
-            f"Error accessing bucket: {bucket}, {e.response['Error']['Message']}"
+        logger.exception(
+            "Error accessing bucket: %s, %s", bucket, e.response["Error"]["Message"]
         )
-        exit
+        return
     for doi_file in s3_client.filter_files_in_bucket(bucket, ".csv", "archived"):
         dois = crossref.get_dois_from_spreadsheet(f"s3://{bucket}/{doi_file}")
         try:
             doi_items = dynamodb_client.retrieve_doi_items_from_database(
                 ctx.obj["doi_table"]
             )
         except ClientError as e:
-            logger.error(f"Table read failed: {e.response['Error']['Message']}")
-            exit
+            logger.exception("Table read failed: %s", e.response["Error"]["Message"])
+            return
         for doi in dois:
             if doi_to_be_added(doi, doi_items):
                 try:
                     dynamodb_client.add_doi_item_to_database(ctx.obj["doi_table"], doi)
                 except ClientError as e:
-                    logger.error(
-                        f"Table error while processing {doi}: {e.response['Error']['Message']}"
+                    logger.exception(
+                        "Table error while processing %s: %s",
+                        doi,
+                        e.response["Error"]["Message"],
                     )
             elif doi_to_be_retried(doi, doi_items) is False:
                 continue
             try:
                 dynamodb_client.update_doi_item_attempts_in_database(
                     ctx.obj["doi_table"], doi
                 )
-            except KeyError as e:
-                logger.error("Key error in table while processing %s: %s", doi, e)
+            except KeyError:
+                logger.exception("Key error in table while processing %s", doi)
             except ClientError as e:
-                logger.error(
-                    f"Table error while processing {doi}: {e.response['Error']['Message']}"
+                logger.exception(
+                    "Table error while processing %s: %s",
+                    doi,
+                    e.response["Error"]["Message"],
                 )
-            crossref_work_record = crossref.get_work_record_from_doi(metadata_url, doi)
-            if crossref.is_valid_response(doi, crossref_work_record) is False:
+            crossref_response = crossref.get_response_from_doi(metadata_url, doi)
+            if crossref.is_valid_response(doi, crossref_response) is False:
```

Comment:
Noticed it here while digging into the refactoring, but seeing it elsewhere as well: we might want to consider our preference for `if x is False:` vs `if not x:` syntax.

Granted that `if not x` will also evaluate for "falsey" values like `0`, `None`, empty lists `[]`, etc., I often find it more readable.  

Don't think it needs changing here, in fact would propose _not_ changing it given the usage elsewhere in the project, but just wanted to put that on our radar as something to look for.